Kempe/Kempe.py

```diff
@@ -2,15 +2,15 @@
 # This code is contributed by Riyazul555
 
 
-# Time Complexity = O(N^3) (approximate)
+# Time Complexity = O(N * M * f(N, M))
```

Comment:
The time complexity should be the same no matter the language used for implementation, so maybe O(2^N)?